website/src/components/pageStateSelectors/SequencingEffortsPageStateSelector.tsx

```diff
@@ -45,6 +51,14 @@ export function SequencingEffortsPageStateSelector({
                     dateRangeFilterConfig={dateRangeFilterConfig}
                 />
             </div>
+            <div>
+                <SelectorHeadline>Variant Filter</SelectorHeadline>
```

Comment:
Do we need this headline? I think on this page it makes more sense to only have "filters", because we don't differentiate the filters in the diagrams anymore.

On the other pages this separation is useful, because the dataset filters are a baseline and the variant filters acts on top of it (e.g. for computing the prevalence over time).
```suggestion
```